### PR TITLE
Fabo/fix node subscriptions

### PR DESCRIPTION
**Pull request by faboweb**
Does the necessary fixes for the new node subscriptions

### CODE REVIEW

**Comment by Bitcoinera** [comment]
Subscription doesn't seem to be working


image

**Comment by Bitcoinera** [comment]
Now subscribing kind of works, aber es stottert noch immer ganz schon viel irgendwie :shrug: 

image

**Comment by Bitcoinera** [comment]
There is a memory leak there:

```
<--- JS stacktrace --->
==== JS stack trace =========================================
    0: ExitFrame [pc: 0x13cb519]
    1: StubFrame [pc: 0x13cc3ed]
Security context: 0x3ea69f6008d1 <JSObject>
    2: encode [0xac9bc23ead9] [/home/ana/lunie/lunie/node_modules/base-x/src/index.js:~25] [pc=0x10d14941ef6e](this=0x0ac9bc23ea41 <Object map = 0x1e67409a29e9>,0x3e10b2dcb201 <Uint8Array map = 0x1df4c8e52d9>)
    3: toJSON [0x35aa05590929] [/home/ana/lunie/lunie/node_modules/@polkadot/types/generic/AccountId.js:~74] [pc=0x10d1492...
FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory
 1: 0xa07f90 node::Abort() [/home/ana/.nvm/versions/node/v12.18.0/bin/node]
 2: 0xa0839c node::OnFatalError(char const*, char const*) [/home/ana/.nvm/versions/node/v12.18.0/bin/node]
 3: 0xb80d9e v8::Utils::ReportOOMFailure(v8::internal::Isolate*, char const*, bool) [/home/ana/.nvm/versions/node/v12.18.0/bin/node]
 4: 0xb81119 v8::internal::V8::FatalProcessOutOfMemory(v8::internal::Isolate*, char const*, bool) [/home/ana/.nvm/versions/node/v12.18.0/bin/node]
 5: 0xd2d875  [/home/ana/.nvm/versions/node/v12.18.0/bin/node]
 6: 0xd2df06 v8::internal::Heap::RecomputeLimits(v8::internal::GarbageCollector) [/home/ana/.nvm/versions/node/v12.18.0/bin/node]
 7: 0xd3a785 v8::internal::Heap::PerformGarbageCollection(v8::internal::GarbageCollector, v8::GCCallbackFlags) [/home/ana/.nvm/versions/node/v12.18.0/bin/node]
 8: 0xd3b635 v8::internal::Heap::CollectGarbage(v8::internal::AllocationSpace, v8::internal::GarbageCollectionReason, v8::GCCallbackFlags) [/home/ana/.nvm/versions/node/v12.18.0/bin/node]
 9: 0xd3e0ec v8::internal::Heap::AllocateRawWithRetryOrFail(int, v8::internal::AllocationType, v8::internal::AllocationOrigin, v8::internal::AllocationAlignment) [/home/ana/.nvm/versions/node/v12.18.0/bin/node]
10: 0xd04cbb v8::internal::Factory::NewFillerObject(int, bool, v8::internal::AllocationType, v8::internal::AllocationOrigin) [/home/ana/.nvm/versions/node/v12.18.0/bin/node]
11: 0x10464be v8::internal::Runtime_AllocateInYoungGeneration(int, unsigned long*, v8::internal::Isolate*) [/home/ana/.nvm/versions/node/v12.18.0/bin/node]
12: 0x13cb519  [/home/ana/.nvm/versions/node/v12.18.0/bin/node]
[nodemon] app crashed - waiting for file changes before starting...
```

**Comment by Bitcoinera** [comment]
Now it is working well! Only one, it seems to get hung up when broadcasting a transaction:

image

File: api/changes/fabo_fix-node-subscriptions [hunk 1]
```diff
@@ -0,0 +1 @@
+[Fixed] Fix new polkadot node subscriptions @faboweb
```
Comment (Bitcoinera) [inline api/changes/fabo_fix-node-subscriptions:1]: Why my change is now gone?

**Review by Bitcoinera — APPROVED** [review]

File: api/changes/fabo_fix-node-subscriptions (hunk 1 above)
Comment (faboweb) [reply api/changes/fabo_fix-node-subscriptions:1]: which change?

Comment (Bitcoinera) [reply api/changes/fabo_fix-node-subscriptions:1]: The one for "return to old polkadot node subscription".

When a change gets rebased by a new bigger change in that file Simsala automatically deletes it or how does it work?

Comment (faboweb) [reply api/changes/fabo_fix-node-subscriptions:1]: no, I reverted it

**Pull request merged**